kvbc/include/categorization/details.h

```diff
@@ -27,7 +27,7 @@ namespace concord::kvbc::categorization::detail {
 
 using Buffer = std::vector<std::uint8_t>;
 
-enum class CATEGORY_TYPE : char { merkle = 0, immutable = 1, kv_hash = 2, end_of_types };
+enum class CATEGORY_TYPE : char { merkle = 0, immutable = 1, versioned_kv = 2, end_of_types };
```

Comment:
Nit: We may want to call this block_merkle, to leave open the option for other merkle tree types.

Reply:
good point